packages/uikit/src/pages/browser/PromotionsCarousel.tsx

```diff
@@ -0,0 +1,53 @@
+import { Body3, Label2 } from '../../components/Text';
+import { FC } from 'react';
+import { Carousel } from '../../components/shared';
+import styled from 'styled-components';
+import { CarouselApp } from '../../hooks/browser/useRecommendations';
+import { PromotedItem, PromotedItemImage, PromotedItemText } from './PromotedItem';
+import { useOpenLinkOnAreaClick } from '../../hooks/useAreaClick';
+
+const CarouselCard = styled.div<{ img: string }>`
+    width: 448px;
+    height: 224px;
+
+    background-image: ${props => `url(${props.img})`};
+    background-size: cover;
+    border-radius: ${props => props.theme.cornerSmall};
+
+    display: inline-flex !important;
+    align-items: flex-end;
+    justify-content: flex-start;
+    cursor: pointer;
+`;
+const CarouselCardFooter = styled(PromotedItem)`
+    margin-left: 1rem;
+`;
+
+export const PromotionsCarousel: FC<{ apps: CarouselApp[]; className?: string }> = ({
```

Comment:
Could you please update carousel layout that it looks good on mobile and do not overlap each other entries?
for example iphone pro max:
   
   
![IMG_285206E8DAA4-1](https://github.com/tonkeeper/tonkeeper-web/assets/22742808/1fdfbf14-c2ba-4415-8ca8-95d614678a5f)

![IMG_5923E8D2AB43-1](https://github.com/tonkeeper/tonkeeper-web/assets/22742808/3c573fb7-13b9-4e4e-b28b-8dd278975b2c)